solr/config/solrconfig.xml

```diff
@@ -117,6 +117,8 @@
           all_text_timv^10
         </str>
 
+        <str name="bq">model_name_ssi:Collection^500.0 </str>
```

Comment:
I don't think you need to include `.0` in there, I think just `^500` is fine. But I'll merge it.